Patterns.Behavioral.State/Bike.cs

```diff
@@ -16,6 +16,16 @@ public Bike(string name)
             State = new RoadBikeState(this);
         }
 
+        public bool AttemptToNegotiatePothole()
+        {
+            return this.State.NegotiatePothole();
+        }
+
+        public void ChangeStateTo(BikeState state)
```

Comment:
Should this be protected so it's only accessible to its subclasses? 🤔  I'm wondering if there's any case where an external party would want to call this

Reply:
It needs to be public because of the `CheckState()` in the different bike state classes which inherit from the `Bike` class.